modules/tinymce/src/plugins/fullscreen/test/ts/browser/FullScreenPluginTest.ts

```diff
@@ -37,7 +37,8 @@ describe('browser.tinymce.plugins.fullscreen.FullScreenPluginTest', () => {
 
   const assertApiAndLastEvent = (editor: Editor, state: boolean) => {
     assert.equal(editor.plugins.fullscreen.isFullscreen(), state, 'Editor isFullscreen state');
-    assert.equal(lastEventArgs.get().state, state, 'FullscreenStateChanged event state');
+    assert.deepEqual(firedEvents, [ 'fullscreenstatechanged', state, 'resizeeditor', state ], 'FullscreenStateChanged event state');
```

Comment:
This label doesn't make sense anymore... maybe something like this?
```suggestion
    assert.deepEqual(firedEvents, [ 'fullscreenstatechanged', state, 'resizeeditor', state ], 'fired event state');
```